CMakeLists.txt

```diff
@@ -97,20 +97,14 @@ set(${PROJECT_NAME}_INSTALL_INCLUDEDIR
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
 
-if (BUILD_WITH_QT6)
-    set(QtMinVersion "6.0")
-else()
-    set(QtMinVersion "5.15")
-    set(QtExtraModules "Multimedia") # See #483
-endif()
-string(REGEX REPLACE "^(.).*" "Qt\\1" Qt ${QtMinVersion}) # makes "Qt5" or "Qt6"
```

Comment:
Let's keep this (and the usages below) - hopefully we'll be able to use it for Qt 7 transition in a few years...